src/modules/vmount/input_test.cpp

```diff
@@ -69,11 +70,16 @@ int InputTest::update(unsigned int timeout_ms, ControlData **control_data, bool
 	_control_data.type_data.angle.frames[1] = ControlData::TypeData::TypeAngle::Frame::AngleAbsoluteFrame;
 	_control_data.type_data.angle.frames[2] = ControlData::TypeData::TypeAngle::Frame::AngleBodyFrame;
 
-	for (int i = 0; i < 3; ++i) {
-		_control_data.type_data.angle.angles[i] = _angles[i] * M_DEG_TO_RAD_F;
+	matrix::Eulerf euler(
+		_angles[0] * M_DEG_TO_RAD_F,
```

Comment:
If you use the mathlib instead we're pretty close to not needing the defines. Whichever you prefer.

```suggestion
		math::radians(_angles[0]),
```

Reply:
Thanks, I'm fixing it all over vmount.